registrations/management/commands/report_pmtct_risks.py

```diff
@@ -115,7 +122,7 @@ def add_to_result(risk, reg):
                 validated=True)
 
             for registration in registrations.iterator():
-
+                add_personally_identifiable_fields(registration)
```

Comment:
Shouldn't we call the task to remove this data again when we're done here? Or would that get called somewhere after this already?